qa/rpc-tests/mempool_limit.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014-2015 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+# Test mempool limiting together/eviction with the wallet
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+class MempoolLimitTest(BitcoinTestFramework):
+
+    def __init__(self):
```

Comment:
This part needs to be refactored between 4 existing tests.
This PRs intent is not to refactor the test framework instead it should add a mempool limit/eviction test.